### PR TITLE
Homework6

### PR DESCRIPTION
**Pull request by litserkovna**

### CODE REVIEW

File: homework6/task2.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    for (let a = num; a >= 0; a--) {
+        console.log(a);
+    }
+}
+countToZero(5);
+function recursiveCounter(num) {
+    if (num > 0) {
+        recursiveCounter(num - 1);
+    }
+}
```
Comment (sttaran) [inline homework6/task2.js]: add console.log to print number on each call

File: homework6/task2.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+function countToZero(num) {
+    for (let a = num; a >= 0; a--) {
+        console.log(a);
+    }
+}
```
Comment (sttaran) [inline homework6/task2.js]: it's not recursion, if you left it for yourself make please a comment for me

**Review by sttaran — CHANGES_REQUESTED** [review]

File: homework6/task2.js (hunk 2 above)
Comment (litserkovna) [reply homework6/task2.js]: Yes, I sorry was checking it and forgot to remove :)

File: homework6/task2.js (hunk 1 above)
Comment (litserkovna) [reply homework6/task2.js]:
```
function recursiveCounter(num) {
console.log(num);
    if (num > 0) {
        recursiveCounter(num - 1);
    }
}
```

**Review by sttaran — APPROVED** [review]

**Pull request merged**